apps/els_lsp/src/els_compiler_diagnostics.erl

```diff
@@ -202,6 +204,436 @@ diagnostic(Range, Module, Desc, Severity) ->
 format_error(Str) ->
   Str.
 
+
+%-----------------------------------------------------------------------
+
+%% @doc Return a unique code for each of the possible errors returned by the
+%% compiler or its subsystems.
+
+-spec make_code(atom(), any()) -> binary().
+
+%% This file
+make_code(els_compiler_diagnostics, _) ->
+ unicode:characters_to_binary("L0000");
```

Comment:
I really don't understand why you are not just returning the binary directly as a literal like this:
```erlang
make_code(els_compiler_diagnostics, _) ->
 <<"L0000">>;
```

Reply:
Good point.